lib/src/map/options/map_options.dart

```diff
@@ -77,13 +54,13 @@ class MapOptions {
   /// Only use this if your map isn't built immediately (like inside FutureBuilder)
   /// and you need to access the controller as soon as the map is built.
   /// Otherwise you can use WidgetsBinding.instance.addPostFrameCallback
-  /// In initState to controll the map before the next frame.
+  /// In initState to control the map before the next frame.
```

Comment:
Might need to change the documentation here to match some of my previous invesigations of bug reports and help questions, and new recommendations on the docs site. Don't worry about that in this PR though.

(PS. thanks for catching my spelling bug below :D)